### PR TITLE
Draft of changelog for 1.1.1-preview release

### PR DESCRIPTION
**Pull request by jennyf19**
Please review and add comments for improving the change log. Thanks.

### CODE REVIEW

**Comment by rayluo** [comment]
The changelog itself looks good to me.

Other than changelog, do we need to bump the old version number somewhere in our code base? It seems that is not applicable, but just double checking. Otherwise we are good to go.

**Comment by jennyf19** [comment]
@rayluo I don't see where we bump up the version in the code base. In ADAL, we use the AssemblyInfo file for that.

**Comment by kpanwar** [comment]
@jennyf19 @rayluo - The OneNote calls out that one should check the current version in nuget.org and compare that to the version in dev branch. dev should already be pointing to vNext.

File: changelog.txt [hunk 1]
```diff
@@ -1,3 +1,14 @@
+1.1.1-preview
+=============
+This release marks the third preview of the library which brings in several features and changes -
```
Comment (kpanwar) [inline changelog.txt]: this is not the third preview. this is a patch release. just say, "This release contrains"

File: changelog.txt [hunk 2]
```diff
@@ -1,3 +1,14 @@
+1.1.1-preview
+=============
+This release marks the third preview of the library which brings in several features and changes -
+- Functionality on Unix with Xamarin.Mac Full profile and Mono (#451)
```
Comment (kpanwar) [inline changelog.txt]: Not sure what this means. This was simply a bug fix where some API call was throwing an exception on mac when running core clr apps

File: changelog.txt [hunk 3]
```diff
@@ -1,3 +1,14 @@
+1.1.1-preview
+=============
+This release marks the third preview of the library which brings in several features and changes -
+- Functionality on Unix with Xamarin.Mac Full profile and Mono (#451)
+- Handle new RSACng default on netfx4.7 (#448)
```
Comment (kpanwar) [inline changelog.txt]: Rephrase this to, "Added support to use RSACng in .net 4.7"

File: changelog.txt [hunk 4]
```diff
@@ -1,3 +1,14 @@
+1.1.1-preview
+=============
+This release marks the third preview of the library which brings in several features and changes -
+- Functionality on Unix with Xamarin.Mac Full profile and Mono (#451)
+- Handle new RSACng default on netfx4.7 (#448)
+- Expose claims as attribute for MSALUiRequiredException (#459)
```
Comment (kpanwar) [inline changelog.txt]: as an*

**Comment by jennyf19** [comment]
@kpanwar i checked Nuget, and it's at 1.1.0-preview. In GitVersion.yml, it says ```next-version: 1.1.1``` is that what you're referring to? Based on what I read on-line the .yml gets updated automatically...is that correct?

File: changelog.txt (hunk 1 above)
Comment (jennyf19) [reply changelog.txt]: done.

File: changelog.txt (hunk 2 above)
Comment (jennyf19) [reply changelog.txt]: Since it's a bug fix, i just removed it

File: changelog.txt (hunk 3 above)
Comment (jennyf19) [reply changelog.txt]: ok thanks
